StarCellBio/settings.py

```diff
@@ -128,16 +128,9 @@
     os.path.join(BASE_DIR, 'StarCellBio', 'templates'),
 ) + auth.settings.TEMPLATE_DIRS
 
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 
 INSTALLED_APPS = (
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.sites',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    # Uncomment the next line to enable the admin:
-    'django.contrib.admin',
     # Uncomment the next line to enable admin documentation:
     'django.contrib.admindocs',
```

Comment:
This doesn't seem to be working, due to a dependency on docutils. Do we still want it?